src/main/haskell/language-kore/src/Data/Kore/Parser/ParserImpl.hs

```diff
@@ -899,6 +898,15 @@ sentenceConstructorRemainderParser sentenceType
                     (symbolParser Object)
                     (Rotate31 <....> SentenceSymbol)
 
+sentenceSortRemainderParser :: Parser KoreSentence
```

Comment:
The Haskell code should stay in sync with the doc, so if the meta sorts are not already in the Sematics of K doc, could you add a Trello task for Xiaohong to add them?